resources/fxaddon/addon.js

```diff
@@ -0,0 +1,30 @@
+(function() {
```

Comment:
Correct. I realized this after submitting the PR but also after I left for the day. I will fix this.